pkg/sysregistriesv2/system_registries_v2.go

```
@@ -275,6 +279,11 @@ func (config *V2RegistriesConf) postProcess() error {
 			}
 		}
 
+		// ignore the credHelper if it is set for inner namespace
+		if reg.CredHelper != "" && strings.Contains(reg.Prefix, "/") {
+			reg.CredHelper = ""
+		}
```

This should report an error instead of silently ignoring invalid input.